examples/CMakeLists.txt

```diff
@@ -4,9 +4,9 @@
 #
 # SPDX-License-Identifier: MIT OR Apache-2.0
 
-# No longer add cargo-without-cmake here.
-# This causes duplicate builds, as this crate is already entirely built
-# When using `cargo test`
-add_subdirectory(qml_features)
 add_subdirectory(qml_minimal)
-add_subdirectory(demo_threading)
+
+if(NOT BUILD_WASM)
+    add_subdirectory(qml_features)
+    add_subdirectory(demo_threading)
+endif()
```

Comment:
ideally we should be building all examples in wasm to ensure that all our features build :-)

Reply:
agree, wanted to start with getting at least one working for this patch, but maybe i should get all?